pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1538,10 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if err := kl.reconcileCbr0(node.Spec.PodCIDR); err != nil {
```

Comment:
I suggested moving it there because in this function we already have a call to Get the current node struct from the master. It seemed silly to be fetching the current node struct in two background loops. I agree that a doc update would be helpful.